packages/deployment-service/src/functions/pipeline/pipeline-tear-down.ts

```diff
@@ -67,14 +67,15 @@ const disableCloudFront = async (Id: string): Promise<any> => {
 }
 
 const tearDownLiveBucketLocation = (location: string): Promise<void> => {
-  return new Promise<void>((resolve, reject) =>
+  return new Promise<void>((resolve) =>
     s3Client.deleteObject(
       {
         Bucket: process.env.DEPLOYMENT_LIVE_BUCKET_NAME as string,
         Key: location,
       },
-      (error) => {
-        error ? reject(error) : resolve()
+      () => {
+        resolve()
+        // error ? reject(error) : resolve()
```

Comment:
idk if u wanna handle the error case here(?)

Reply:
yeaaaa not sure I wanna keep it. I'll try and get around it and see if it was just distro issues